## Biography

As [informatics manager](https://www.imperial.ac.uk/people/jake.pearce) for the [Division of Integrative Systems Medicine and Digestive Disease](https://www.imperial.ac.uk/department-surgery-cancer/research/integrative_systems_medicine/), I lead the teams responsible for development and implementation of the research software and databases that underpin the Division's research.

Additionally I lead the teams in the [MRC-NIHR National Phenome Centre](https://phenomecentre.org), and [Imperial BRC Clinical Phenotyping Centre](http://www.imperial.ac.uk/clinical-phenotyping-centre), responsible for the statistical and computational aspects of both centres data-acquisition programs, monitoring the ongoing quality control, maintaining the laboratory information systems that manage the work flow, and conducting statistical analyses of the datasets generated.

My previous positions include the EU FloriNASH project, investigating the link between gut microflora and the progression from normal function to non-alcoholic fatty-liver disease (NAFLD) and non-alcoholic steatosis (NASH) in clinical patients. My work involved the profiling of the structure of clinical cohorts, and integration of metabolite-profiles, transcriptomic and proteomic datasets.

While studying for my PhD in Computational Medicine, titled _Novel Computational Approaches to Characterising Metabolic Responses to Toxicity_ via _an NMR-based Metabonomic Database_ (pdf available of [my PhD](https://www.imperial.ac.uk/people/jake.pearce/document/2363/JTMPearce_Thesis/?JTMPearce_Thesis.pdf)), I also took the role of Data-Curator for the Consortium for Metabonomic Toxicology (COMET) project. In this capacity, I ensured the quality of the processed data as well as designing and implementing software to assist in the analysis of the COMET dataset.

Previous to my time at Imperial College, I studied Biochemistry at the University of Sussex. While at Sussex I undertook an undergraduate project in X-ray crystallography, and succeeded in growing the first X-ray quality crystal of the human protein Phosphoglucosse Isomerase, the structure of which I assisted in the elucidation of, and which is available from the PDB with the code [1IAT](https://www.rcsb.org/structure/1iat).

Upon completing my BSc I undertook an MSc in Bioinformatics at Birkbeck College, London. During my time at Birkbeck my project work specialised in database manipulation and integration, with special reference to the Gene Ontology database and FlyBase genome repository.

### Quicklook Plugins for macOS
My work has involved extensive use of MATLAB for computational analysis of metabolic profiling data-sets. To this end I have wrote a pair of plug-ins for macOS to improve the integration of MATLAB with the operating system.

The [MATLAB quicklook generator](https://github.com/jaketmp/matlab-quicklook) (for macOS 10.6 Snow-Leopard and later) extends quicklook to give an overview of the variables in a .mat workspace file.

The metadata importer (for Mac OS X 10.4 and later) allows Spotlight to index the names of variable within a .mat file.

Both plug-ins are now updated to run in 64bit under Mac OS X 10.6 and later, and included several speed optimisations.

Additionally I have written a [quicklook plugin](https://github.com/jaketmp/ePub-quicklook) to preview [ePub documents](http://idpf.org/epub). Downloads and source code for both plugins are available at GitHub.

### International Phenome Centre Network
Working as part of the [International Phenome Centre Network](http://phenomenetwork.org/) (IPCN), I am interested in the challenges inherent in comparing healthcare data from diverse systems around the world, leveraging the power that can be gained both by analysing this data in concert and validating findings in totally independent populations.
